src/Dashboard/View.purs

```diff
@@ -99,7 +101,7 @@ formatStatus { id: PipelineId id, status } =
     ]
     [ H.text $ "#" <> show id
     , H.br []
-    , H.text $ String.toUpper $ show status
+    , H.text $ capitalize $ show status
```

Comment:
No need for `capitalize` really, data constructors are uppercased already and that's what `show` uses.